### ISSUE TITLE
Way to unlock bootloader has changed since security update A525FXXS4BVI3

### ISSUE THREAD

**Issue by powerman** [Simon1511/android_device_samsung_a52q#1]
Now it works this way:

> With the device booted and screen unlocked hold <kbd>Volume Up</kbd> + <kbd>Volume Down</kbd> + <kbd>Power</kbd> while the device is connected to a PC via USB cable.

After booting it'll show instructions (hold <kbd>Volume Up</kbd> to enter unlock OEM mode and then press <kbd>Volume Up</kbd> to confirm).

Please update LineageOS Wiki.

**Comment by Simon1511** [comment]
> With the device booted

So you're saying you now need to be booted into Android to unlock your phone?

This is the first time I hear about this, also I can't seem to find anything regarding that on the Internet.

**Comment by powerman** [comment]
Yeah. Just did that. I've found this info on "russian xda" https://4pda.to/forum/index.php?s=&showtopic=1021105&view=findpost&p=118165336 but author says original information is from https://forum.xda-developers.com/.

**Comment by powerman** [comment]
Just boot into LineageOS recovery. So, I suppose, rest of instructions are still actual.

**Comment by Simon1511** [comment]
Hmm, interesting.

I'll see if I can confirm this.

**Comment by powerman** [comment]
The funny thing is it works this way even in LineageOS.

**Comment by Simon1511** [comment]
I just checked together with someone else, he installed the latest A525FXXS4BVI3 (India CSC) and had no problems.
The keycombos worked the same way for him as they worked before.

I'll see if I have time to check on my own phone too, but thats probably not gonna happen in the next few weeks.
